src/components/FamilyManifestModal/FamilyDataTypesStatsQuery.js

```diff
@@ -60,8 +60,8 @@ export default compose(withApi)(
       }, {})}
       render={({ data, loading }) => {
         return render(
-          loading
-            ? { loading, data: [] }
+          loading || !data
+            ? { loading: true, data: [] }
```

Comment:
this seems odd.. if this `loading` prop is driving a spinner this would lead to a seemingly infinite load, my guess is that that's not actually what we want?